networking/robot.py

```diff
@@ -0,0 +1,64 @@
+import lcm
+import threading
+import time
+from config import Commands as cmd
+
+class Robot():
+  """
+  Basic robot simulator. Features
+  game states as well as estop and basic i/o
+  """
+  def __init__(self, number):
+    self.number = number
+    self.teleop = False
+    self.motors = [0]*4
+    self.sensors = [True, 4]
+    self.enabled = False
+    self.estop = False
+        
+def heartbeat(lc, number):
+  """
+  Generates heartbeats for the robot
+  sends robot number as well as state flag
+  (0 = disabled, 1 = autonomous, 2 = teleop, 3 = estop)
+  """
+  i = 0
+  while True:
+    i += 1
+    state = robot.enabled + robot.teleop + robot.estop*2
+    status = "{} {}".format(robot.number, state)
+    lc.publish("ROBOT_HEARTBEAT", status)
```

Comment:
I notice you're publishing strings on LCM.

I strongly recommend that you switch to publishing using LCM types instead (or another strict encodingy system like protobuf) -- using types helps you keep your sanity when building a distributed system.